README.md

```diff
@@ -1 +1,10 @@
 # Git 101 Workshop - Spirit Animals!
+
+The challenge is to add a line to this file (in order!) with your name and your spirit animal.
+
+1. Steve  - My  spirit animal is the goat, because it will eat anything.
+2. Chad - My spirit animal is a cat, because I did a quiz while Steve was presenting.
+4. Yasemin - My spririt animal is a bird, because it will fly anywhere it wants.
+5. Vikki - My spirit animal is a cat, because they like to sleep and eat lots.
+10. Urszula - My spirit animal is a bear, bacause I like honney.
+
```

Comment:
11. Andreas- My spirit animal is a badger, because they are gruff-but-benevolent souls.